### ISSUE TITLE
MS Sans Serif font is "squished" when bold (Xubuntu 20.04)

### ISSUE THREAD

**Issue by nullnothing** [grassmunk/Chicago95#192]
Hi all. I talked about this in #140 , but I figured that this "squishing" issue probably deserves its own, seperate issue post.

After following the instructions for installing the Microsoft Sans Serif font in Xubuntu 20.04, the regular font seems to work fine. However, the bold font becomes noticeably squished.

(Ignore the title font comment, just focus on the bold "Appearance" header text)
![2](https://user-images.githubusercontent.com/47261624/122518161-35994700-d054-11eb-8e17-cb1aa002f3e8.png)

Comparing the bold fonts between the two setups in the image above, it becomes obvious that the bold font's characters are spaced too close in Xubuntu 20.04,which looks wrong and sometimes unreadable. From my (somewhat limited) testing, this appears to happen regardless of the font size. I'll verify this when I get the time to set up another VM and test it properly.

I'm not too informed when it comes to fonts and such, so I don't know what else to say about the issue. If you need more information, please don't hesitate to ask. 

To reproduce:
- Create a fresh install of Xubuntu 20.04
- Install Chicago95 with the `installer.py` script
- Follow the instructions in `INSTALL.md` to install the MS Sans Serif font
- Open the XFCE Appearance settings menu (although I think anywhere that tries to "embolden" the MS Sans Serif font will display this issue)
- Observe as the emboldened MS Sans Serif font has characters spaced too close, making it look squished

**Comment by nullnothing** [comment]
I started to do some testing on the issue, and so far, this appears to happen only if [these changes](https://github.com/grassmunk/Chicago95/issues/140#issuecomment-718978959) are applied to the font configurations, which is unfortunately the only way to have a bitmap titlebar, too. As such, this issue is most likely directly related to the titlebar font issue (#140 ), so I guess this issue post is effectively useless.

I guess I should close this issue. I did manage to find a way to get the title font to render like a bitmap while testing, too, although I'm not sure exactly what I did. I'll do some more testing and post what I find over on #140 .
